tests/unit/src/test/scala/tests/MetalsEnrichmentsSuite.scala

```diff
@@ -24,6 +24,17 @@ class MetalsEnrichmentsSuite extends BaseSuite {
 
   // windows doesn't %20 as escapes
   if (!isWindows) {
+    test("dont decode nor encode") {
+      val jvmManager = coursierapi.JvmManager.create()
```

Comment:
Yes it will.

> Could we instead just use an example string?
No we can't because `srcUri.toAbsolutePath` open the jar file system. So at least I need a jar file. Maybe it would work on an empty jar file. I am trying that.